src/parsers/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# ---------------------------------------------------------------------------- #
+# ~~~ Set target names
+set(_LIB_OUTPUT_NAME   "parsers")
+set(_TEST_OUTPUT_NAME  "test")
+
+set(HUNSPELL_PARSERS_TARGET_PREFIX  "parsers"                                                 CACHE INTERNAL  "")
+set(HUNSPELL_PARSERS_LIB_TARGET     "${HUNSPELL_PARSERS_TARGET_PREFIX}_lib"                   CACHE INTERNAL  "")
```

Comment:
What's with the suffix? See comment above about idiomatic naming on different platforms.